docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+services:
+  postgresql:
+    build:
+      context: .
+      dockerfile: Dockerfile.postgresql
+    ports:
+      - "5432:5432"
```

Comment:
`127.0.0.1:5432:5432` seems to work fine here (on docker for mac at least) and avoids a network peer being potentially able to connect to the postgres (which is a quite wide trust profile).

So let's do that (and probably do it for safecastapi as well)